proposals/exp-show-target.md

```diff
@@ -0,0 +1,214 @@
+│ 8bf2091        │ Nov 15, 2020 │       - │ 0.2           │ 20170428     │ 500                    │ 1                │ 20170428   │ 50                 │ -                  │ - │
+│ ff9e2fa        │ Nov 15, 2020 │       - │ 0.2           │ 20170428     │ 500                    │ 1                │ 20170428   │ 50                 │ -                  │ - │
+│ 11e1705        │ Nov 15, 2020 │       - │ 0.2           │ 20170428     │ 1500                   │ 2                │ 20170428   │ 10                 │ -                  │ - │
+│ a82585e        │ Nov 15, 2020 │       - │ 0.2           │ 20170428     │ 1500                   │ 2                │ 20170428   │ 10                 │ -                  │ - │
+│ c778a78        │ Nov 15, 2020 │       - │ 0.2           │ 20170428     │ 1500                   │ 2                │ 20170428   │ 10                 │ -                  │ - │
+│ 551082e        │ Nov 15, 2020 │       - │ 0.2           │ 20170428     │ 1500                   │ 2                │ 20170428   │ 10                 │ -                  │ - │
+│ 15bef96        │ Nov 15, 2020 │       - │ 0.2           │ 20170428     │ 1500                   │ 2                │ 20170428   │ 10                 │ -                  │ - │
+└────────────────┴──────────────┴─────────┴───────────────┴──────────────┴────────────────────────┴──────────────────┴────────────┴────────────────────┴────────────────────┴───┘
+```
+
+This table includes parameters and metrics from the `evaluate` stage and all
+stages upstream of it.  It's possible to also have a `--single-item` argument to
+exclude upstream changes, but this is out of scope for now.
+
+# How We Teach This
+
+See [Detailed design](#detailed-design) for a proposed help message.
+
+This is consistent with other commands, and it's only one additional argument to
+an existing command, so it likely doesn't require much teaching. It might be
+necessary to explicitly add that parameters and metrics from upstream stages
+will be shown.
+
+# Drawbacks
+
+This proposal may require changes to how DVC collects and tracks parameters and
+metrics. Implementation could break existing functionality, and it is
+inconsistent with the current behaviors of `dvc params`, `dvc metrics`, and `dvc
+exp diff` commands.
+
+Further, this proposal adds to an already crowded set of `dvc exp show` options,
+and there is no single way to show parameters and metrics that's going to work
+for every use case. For example, this proposal does nothing to filter out
+parameters for upstream stages, which may frequently be irrelevant if only the
+stages near the end of the pipeline change during experimentation.
+
+# Alternatives
+
+There are many alternative methods to refine the output of `dvc exp show`:
+
+* Current functionality: `exp show` already has options to include/exclude any
+  column, so the flexibility to get the same output as suggested here already
+  exists.
```

Comment:
But DVC still has to read all those stages and values whereas with `targets` the operation could actually improve performance-wise.